README.md

```diff
@@ -1 +1,18 @@
 # Task-Tracker-App
+
+The app is designed to work with the [Task-Tracker-Device](https://github.com/Task-Tracker-Systems/Task-Tracker-Device).
+
+## Build
+
+### Local
+Run the gradle build in each directory or use the `build.sh` file.
+
+### Docker
+Powershell
+```Powershell
+docker build -t task-tracker-systems/task-tracker-app:1 .
+docker run --rm -v ${PWD}:/dev/mnt task-tracker-systems/task-tracker-app:1
```

Comment:
Where is the output of the build? Executing this line, the output ends with 
```
5 actionable tasks: 4 executed, 1 up-to-date
```

Reply:
it should be in the `build` directory on the host, do you find it there?

Reply:
The 'host' is the system which hosts Docker?

After executing this (and the previous) line I don't have a `./build/` directory in the root directory. But there are 31 nested `build` directories.

Is the output to be executed by the user, this file?

    ./Task-Tracker-CLI/build/scripts/Task-Tracker-CLI